modules/navier_stokes/doc/content/source/fvkernels/WCNSFV2PInterfaceAreaSourceSink.md

```diff
@@ -0,0 +1,114 @@
+# WCNSFV2PInterfaceAreaSourceSink
+
+The interfacial area concentration is defined as the interface area between
+two phases per unit volume, i.e.,$[\xi_p] = \frac{m^2}{m^3}$. This parameter is important
+for predicting mass, momentum, and energy transfer at interfaces in two-phase flows.
+
+The general equation for interfacial area concentration transport via the mixture model reads as follows:
+
+\begin{equation}
+\frac{\partial (\rho_d \xi_p)}{\partial t} + \nabla \cdot \left( \rho_d \vec{u} \xi_p \right) - \nabla \left( D_b \nabla \xi_p \right) =
+-\frac{1}{3} \frac{D \rho_d}{Dt} + S_T + \rho_d (S_C + S_B)\,,
+\end{equation}
+
```

Comment:
I didn't mark the right place for density term missing. Here is what I mean, the diffusion term here has a unit discrepancy.

Reply:
Ok, I thought the unit is m2/s. Then it should be fine.